README.md

```diff
@@ -136,6 +136,76 @@ custom:
   datadog:
     enabled: ${self:custom.staged.dd_enabled, true}
 ```
+### Serverless Monitors 
```

Comment:
I would add the new `monitorsApiKey` and `monitorsAppKey` options to the list of parameters, and make sure that the differences in behavior between `monitorsApiKey` and `apiKey` are clear.